egs/voxceleb/v2/rerun_with_meld.sh

```diff
@@ -0,0 +1,165 @@
+
+  # Prepare the MUSAN corpus, which consists of music, speech, and noise
+  # suitable for augmentation.
+  steps/data/make_musan.sh --sampling-rate 16000 $musan_root ${data_dir}
+
+  # Get the duration of the MUSAN recordings.  This will be used by the
+  # script augment_data_dir.py.
+  for name in speech noise music; do
+    utils/data/get_utt2dur.sh ${data_dir}/musan_${name}
+    mv ${data_dir}/musan_${name}/utt2dur ${data_dir}/musan_${name}/reco2dur
+  done
+
+  # Augment with musan_noise
+  steps/data/augment_data_dir.py --utt-suffix "noise" --fg-interval 1 --fg-snrs "15:10:5:0" --fg-noise-dir "data/musan_noise" ${data_dir}/train ${data_dir}/train_noise
+  # Augment with musan_music
+  steps/data/augment_data_dir.py --utt-suffix "music" --bg-snrs "15:10:8:5" --num-bg-noises "1" --bg-noise-dir "data/musan_music" ${data_dir}/train ${data_dir}/train_music
+  # Augment with musan_speech
+  steps/data/augment_data_dir.py --utt-suffix "babble" --bg-snrs "20:17:15:13" --num-bg-noises "3:4:5:6:7" --bg-noise-dir "data/musan_speech" ${data_dir}/train ${data_dir}/train_babble
+
+  # Combine reverb, noise, music, and babble into one directory.
+  utils/combine_data.sh ${data_dir}/train_aug ${data_dir}/train_reverb ${data_dir}/train_noise ${data_dir}/train_music ${data_dir}/train_babble
+fi
+
+if [ $stage -eq 5 ]; then
+  # Take a random subset of the augmentations
+  utils/subset_data_dir.sh ${data_dir}/train_aug 10000 ${data_dir}/train_aug_1m
+  utils/fix_data_dir.sh ${data_dir}/train_aug_1m
+
+  # Make MFCCs for the augmented data.  Note that we do not compute a new
+  # vad.scp file here.  Instead, we use the vad.scp from the clean version of
+  # the list.
+  steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 40 --cmd "$train_cmd" \
+    ${data_dir}/train_aug_1m ${root}/exp/make_mfcc $mfccdir
+
+  # Combine the clean and augmented MELD list.  This is now roughly
+  # double the size of the original clean list.
+  utils/combine_data.sh ${data_dir}/train_combined ${data_dir}/train_aug_1m ${data_dir}/train
+fi
+
+# Now we prepare the features to generate examples for xvector training.
+if [ $stage -eq 6 ]; then
+  # This script applies CMVN and removes nonspeech frames.  Note that this is somewhat
+  # wasteful, as it roughly doubles the amount of training data on disk.  After
+  # creating training examples, this can be removed.
+  local/nnet3/xvector/prepare_feats_for_egs.sh --nj 2 --cmd "$train_cmd" \
+    ${data_dir}/train_combined ${data_dir}/train_combined_no_sil ${root}/exp/train_combined_no_sil
+  utils/fix_data_dir.sh ${data_dir}/train_combined_no_sil
+fi
+
+# ./run.sh does a bunch of filtering of utterances by speakers
+# that are too infrequent -- we skip that here speaker=emotion label 
+if [ $stage -eq 7 ]; then
+  # Now, we need to remove features that are too short after removing silence
+  # frames.  We want atleast ~1s (100 frames) per utterance. (note this is smaller than in v2/run.sh)
+  min_len=100
```

Comment:
note that this is a change from the origin v2/run.sh script which filters anything shorter than 5s.  as our model is already partially trained off Vox and we don't have a ton of labeled emotion data, I allow shorter MELD segments to remain part of our training set.